### PR TITLE
Fix the wrong literal of timestamp-format

### PR DESCRIPTION
**Pull request by Al-assad**
Signed-off-by: Al-assad yulin.ying@outlook.com

According to the [documentation](https://github.com/tidb-incubator/TiBigData/tree/master/flink#configuration), the special time formatting configuration should be `timestamp-format.${columnName}` instead of `tidb.timestamp-format.${columnName}`

### CODE REVIEW

**Comment by sunxiaoguang** [comment]
It was intentional to rename properties with common tidb prefix like other properties of TiDB connector and builtin connectors of flink. 

Could you make a change in README.md to document this change so people know there is such change. And in the old version of connector we can change to read from both new properties with and without prefix. This way people won't be bothered during migration.

**Comment by sunxiaoguang** [comment]
There are still some properties without prefix, they were copied from JDBC connector for table sink. When we have builtin global transaction support, all these properties are going to be replaced by different set of configurations. Maybe you can redirect description of those properties to Flink document in README.md.

**Comment by Al-assad** [comment]
> There are still some properties without prefix, they were copied from JDBC connector for table sink. When we have builtin global transaction support, all these properties are going to be replaced by different set of configurations. Maybe you can redirect description of those properties to Flink document in README.md.

Thanks for your reply, I agree with you that is important to keep the key configuration items compatible in different version, actually i was confused about this recently when writing relevant unit tests.
I'll update this PR with the relevant compatibility code later.

**Comment by sunxiaoguang** [comment]
> > There are still some properties without prefix, they were copied from JDBC connector for table sink. When we have builtin global transaction support, all these properties are going to be replaced by different set of configurations. Maybe you can redirect description of those properties to Flink document in README.md.
> 
> Thanks for your reply, I agree with you that is important to keep the key configuration items compatible in different version, actually i was confused about this recently when writing relevant unit tests.
> I'll update this PR with the relevant compatibility code later.

In case I didn't make it clear let me rephrase it. We can remove old property in 1.13 and mark old properties deprecated in document and support reading from both properties in 1.11 and 1.12. This project is still in early adoption stage, as long as we make it clear in document, people would like to eliminate debts asap.

**Comment by Al-assad** [comment]
with you on that @sunxiaoguang
